### ISSUE TITLE
(0) Source error in release and debug mode

### ISSUE THREAD

**Issue by NiharDodiya** [ryanheise/just_audio#405]
NOTE! This is probably related to issue [#395](https://github.com/ryanheise/just_audio/issues/395).

**Which API doesn't behave as documented, and how does it misbehave?**
I guess it's AudioPlayer.play() that isn't behaving as expected.

**Minimal reproduction project**
No idea

**To Reproduce (i.e. user steps, not code)**
1. initialise and start audio player

**Error messages**

```
I/flutter (20654): (0) Source error
```

**Expected behavior**
No Crash

**Screenshots**
not require 

**Desktop (please complete the following information):**
 not tried 

**Smartphone (please complete the following information):**
 - Device: Vivo 1801
 - OS: Android

**Flutter SDK version**
```
SDK 2.0.6 stable
```
**Additional context**
No

**Comment by ryanheise** [comment]
How can I reproduce this?

**Comment by NiharDodiya** [comment]
IN my case i'm getting this error in below code

```
  try{
    await _player.setAudioSource(_concatenatingAudioSource);
    await _player.setLoopMode(LoopMode.one);
    await _player.load();
    await onPlay();
  }catch(e){
  print(e);
}
```

**Comment by ryanheise** [comment]
I'm going to close this because it is the 4th issue repost that you've made without providing the minimal reproduction project. If you don't provide a reproduction project, I can't investigate it and fix it.

**Comment by NiharDodiya** [comment]
I dont get it, What should i have to provide to you?

**Comment by ryanheise** [comment]
I've provided instructions right in the issue template under the section "Minimal reproduction project"
 - I guess the instructions were not clear. I will improve the instructions, but can you quote which part of the instructions you find unclear?

**Comment by NiharDodiya** [comment]
I actually dont get the clear meaning of Minimal reproduction project

**Comment by ryanheise** [comment]
Ok but don't the instructions below the section title at least explain what you need to do? Do you understand the instructions? If not, which part of the instructions are confusing?

**Comment by NickSereda** [comment]
I'm having the same issue, seems like it can't make valid assets play

**Comment by NickSereda** [comment]
debugging shows that "await _player.setAudioSource(_concatenatingAudioSource);" might be the problem

**Comment by ryanheise** [comment]
Please open a new issue. This will allow you to fill out the form with the information I require to investigate the bug

**Comment by NickSereda** [comment]
I have found where the issue was, the issue was that assets with paths that contain spaces " " sometimes cause this Source error (0)

**Comment by ryanheise** [comment]
This was addressed in #325 but if there is another issue it can be fixed if you open a new issue.
